src/bussinessLogic/ManageArray.java

```diff
@@ -16,6 +18,8 @@
     // hash map for storing the number of times a char is displayed (needed for
     // calculating the average)
     private final LinkedHashMap<String, Integer> dividerMap = new LinkedHashMap<>();
```

Comment:
Eclipse says that this field's value is not used.

Reply:
True. I moved it in _ManageMap_ and I forgot to delete it.